stories/Notification.stories.ts

```diff
@@ -0,0 +1,56 @@
+import { Story, Meta } from '@storybook/angular/types-6-0';
+import { NotificationComponent } from '../projects/ion/src/lib/notification/notification.component';
+import { moduleMetadata } from '@storybook/angular';
+import { IonIconComponent } from '../projects/ion/src/lib/icon/icon.component';
+
+export default {
+  title: 'Ion/Feedback/Notification',
+  component: NotificationComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [],
+      declarations: [IonIconComponent],
+    }),
+  ],
+} as Meta;
+
+const Template: Story<NotificationComponent> = (
```

Comment:
It would be interesting to add a behavior about the display time of the notification in Storybook, because as it is currently, the notification is being "closed" with an automatic time and often not being able to see it.